spec/implicit_conversion_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
```

Comment:
Do these tests cover all of the changes? Maybe something to consider is to run all existing tests but with wrappers too - if there's a least intrusive way of doing this.

Reply:
I've added CI back to this project and merged those changes in.    I've also ported the compliance tests into the implicit conversion test cases - there were a few failures in functions which I've fixed. 

There are a few other failures in the compliance tests w/ implicit conversion that I'm not positive how to address (they may be issues in my spec code as well) - could you take a look at those cases?